helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -354,10 +359,10 @@ public void enqueueTask(NotificationContext changeContext) throws Exception {
     // async mode only applicable to CALLBACK from ZK, During INIT and FINALIZE invoke the
     // callback's immediately.
     if (_batchModeEnabled && changeContext.getType() == NotificationContext.Type.CALLBACK) {
-      logger.debug("Enqueuing callback");
+      logger.debug("Callbackhandler {}, Enqueuing callback", this._uid );
       if (!isReady()) {
-        logger.info("CallbackHandler is not ready, ignore change callback from path: {}, for "
-            + "listener: {}", _path, _listener);
+        logger.info("CallbackHandler {} is not ready, ignore change callback from path: {}, for "
+            + "listener: {}", this._uid, _path, _listener);
```

Comment:
nit, "this." in "this._uid" is not necessary, right?

Reply:
removed all this.